python/lsst/daf/butler/registry/interfaces/_database.py

```diff
@@ -136,7 +137,8 @@ def addTableTuple(self, specs: Tuple[ddl.TableSpec, ...]) -> Tuple[sqlalchemy.sc
         is just a factory for `type` objects, not an actual type itself,
         we cannot represent this with type annotations.
         """
-        return specs._make(self.addTable(name, spec) for name, spec in zip(specs._fields, specs))
+        return specs._make(self.addTable(name, spec)                     # type: ignore
+                           for name, spec in zip(specs._fields, specs))  # type: ignore
```

Comment:
I feel like there's probably something better than `# type: ignore`, but `collections.namedtuple` is very weird in the type system, and I couldn't come up with anything better either.